example/burrow_integration/asset.py

```diff
@@ -0,0 +1,258 @@
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    return hex_hash
+
+
+@trace
+def get_asset(address):
+    params = get_first_four_bytes_of_keccak(b"getAsset(string)")
+    no_of_param = 1
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding("cc#burrow")  # asset id
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    return hex_hash
+
+
+@trace
+def add_asset(address):
+    params = get_first_four_bytes_of_keccak(b"addAsset(string,string)")
+    no_of_param = 2
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding("cc#burrow")  # asset id
+    params = params + argument_encoding("500")  # amount of asset
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+
+
+@trace
+def subtract_asset(address):
+    params = get_first_four_bytes_of_keccak(b"subtractAsset(string,string)")
+    no_of_param = 2
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding("cc#burrow")  # asset id
+    params = params + argument_encoding("300")  # amount of asset
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+
+
+@trace
+def balance(address):
+    params = get_first_four_bytes_of_keccak(b"queryBalance(string,string)")
+    no_of_param = 2
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding(ADMIN_ACCOUNT_ID)  # account id
+    params = params + argument_encoding("cc#burrow")  # asset id
```

Comment:
I suggest to add global constant e.g.:
`ASSET_ID` instead of the same text in multiple places